### DISCUSSION TITLE
카테고리의 다른 글 중복

### DISCUSSION THREAD

**Discussion by MJKim99** [pronist/hELLO#151] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: Windows 10
- 브라우저: Chrome 107.0.5304.122
- 스킨 버전 3.6.5

**설명**
스킨편집에 있는 카테고리의 다른 글 말고도, 또다른 카테고리의 다른 글 영역이 노출되고 있습니다.
없앨 수 있는 방법이 있을까 문의드립니다.

**스크린샷 및 링크**
![image](https://user-images.githubusercontent.com/81893393/204036230-ef464a6c-86c5-46fa-88e8-71a4a1f18b82.png)

https://mimah.tistory.com/entry/Thymeleaf-JavaScript%EC%97%90%EC%84%9C-%ED%83%80%EC%9E%84-%EB%A6%AC%ED%94%84-%EB%B3%80%EC%88%98-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0

**Comment by pronist** [comment, marked as answer]
@MJKim99 안녕하세요. **관리자 - 플러그인**으로 가셔서 `카테고리 글 더 보기` 티스토리 플러그인을 해제해 주세요.

![1](https://user-images.githubusercontent.com/25624965/204082814-a1cc4cf8-49a2-4262-8c80-59adc25c3be3.png)
